sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -3390,13 +3390,14 @@ object SQLConf {
       .booleanConf
       .createWithDefault(true)
 
-  val REPLACE_NULLIF_USING_WITH_EXPR =
-    buildConf("spark.databricks.sql.replaceNullIfUsingWithExpr")
```

Comment:
Wow, was this `spark.databricks` before?

Reply:
my bad, i forgot to remove that in my previous PR

Reply:
:-).